crates/biome_css_formatter/src/utils/component_value_list.rs

```diff
@@ -160,6 +168,16 @@ pub(crate) enum ValueListLayout {
     ///     sans-serif;
     /// ```
     OnePerLine,
+
+    /// Separate values by comma into multiple groups, and print each group on a single line
```

Comment:
In the PR description, you explained when this layout was applied. That is **essential knowledge** and we want to keep it around, so please add that information here :)